client/src/catalog-element-readme.html

```diff
@@ -230,6 +226,11 @@
         links = Polymer.dom(this.$.contents).querySelectorAll('[src]');
         for (i = 0; i < links.length; i++)
           links[i].setAttribute('src', links[i].getAttribute('src').replace(relativeRegex, githubBase + '/raw/' + defaultBranch + '/$1'));
+
+        // Hide first heading if its the same as the repo name.
+        var firstHeading = this.$.contents.querySelector('h1, h2');
+        if (firstHeading && firstHeading.textContent.match(this.data.repo))
```

Comment:
Should this just be `includes` or `indexOf` instead of `match`? I assume `this.data.repo` is not intended to be a regexp.